src/neo-vm/Types/Null.cs

```diff
@@ -14,19 +14,14 @@ public override bool Equals(StackItem other)
             return false;
         }
 
-        public override bool GetBoolean()
-        {
-            return false;
-        }
-
-        public override ReadOnlySpan<byte> GetByteArray()
+        public override int GetHashCode()
         {
             throw new NotSupportedException();
```

Comment:
From https://docs.microsoft.com/en-us/dotnet/api/system.object.gethashcode " The GetHashCode() method should not throw exceptions"

Reply:
Maybe is better to return 0

Reply:
In my opinion, `GetHashCode()` is used only if the object is used as a key of a hash table. (Such as `Dictionary`, or `HashSet`.) That's why if we override `Equals()`, we must override `GetHashCode()`. But in NeoVM, `StackItem` should not be a key of `Dictionary` except `PrimitiveType`. So we throw `NotSupportedException` to avoid this case.

Reply:
And also, I didn't change the behavior of `StackItem.GetHashCode()` in this PR.